<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBook</class>
 <widget class="QMainWindow" name="PhoneBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>دفترچه تلفن</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLineEdit" name="searchEdit">
      <property name="placeholderText">
       <string>جستجوی مخاطب...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPushButton" name="searchBtn">
      <property name="text">
       <string>جستجو</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="contactsList"/>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="placeholderText">
         <string>نام مخاطب</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addContactBtn">
        <property name="text">
         <string>اضافه کردن مخاطب</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editContactBtn">
        <property name="text">
         <string>ویرایش مخاطب</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteContactBtn">
        <property name="text">
         <string>حذف مخاطب</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneTypeEdit">
        <property name="placeholderText">
         <string>نوع شماره (مثل: خانه، موبایل، کار)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneNumberEdit">
        <property name="placeholderText">
         <string>شماره تلفن</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPhoneBtn">
        <property name="text">
         <string>اضافه کردن شماره</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editPhoneBtn">
        <property name="text">
         <string>ویرایش شماره</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePhoneBtn">
        <property name="text">
         <string>حذف شماره</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="phoneNumbersList"/>
      </item>
      <item>
       <widget class="QListWidget" name="favoritesList"/>
      </item>
      <item>
       <widget class="QPushButton" name="addFavoriteBtn">
        <property name="text">
         <string>اضافه به علاقه‌مندی‌ها</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeFavoriteBtn">
        <property name="text">
         <string>حذف از علاقه‌مندی‌ها</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveUpBtn">
        <property name="text">
         <string>جابجا به بالا</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveDownBtn">
        <property name="text">
         <string>جابجا به پایین</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="saveBtn">
      <property name="text">
       <string>ذخیره (Ctrl+S)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="loadBtn">
      <property name="text">
       <string>بارگذاری (Ctrl+L)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="exportBtn">
      <property name="text">
       <string>صادر کردن (Ctrl+E)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="importBtn">
      <property name="text">
       <string>وارد کردن (Ctrl+I)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>فایل</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>ذخیره</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>بارگذاری</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>صادر کردن</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>وارد کردن</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>